src/main/kotlin/no/nav/syfo/fastlege/ws/adresseregister/AdresseregisterClient.kt

```diff
@@ -72,6 +85,38 @@ class AdresseregisterClient(
             throw e
         }
 
+    private fun extractPhysicalAddress(wsOrg: WSOrganization, field: String) =
+        wsOrg.physicalAddresses?.physicalAddresses?.stream()
+            ?.filter { wsAddress ->
+                wsAddress.type != null &&
+                    wsAddress.type.isActive &&
+                    wsAddress.type.codeValue == field
+            }
+            ?.findFirst()
+            ?.map { wsAddress ->
+                Adresse(
+                    adresse = if (wsAddress.postbox.isNullOrBlank()) wsAddress.streetAddress else wsAddress.postbox,
+                    postnummer = postnummer(wsAddress.postalCode),
+                    poststed = wsAddress.city,
+                )
+            }
+            ?.orElse(null)
+
+    private fun extractElectronicAddressField(wsOrg: WSOrganization, field: String) =
+        wsOrg.electronicAddresses?.electronicAddresses?.stream()
+            ?.filter { it.type.codeValue == field }
+            ?.findFirst()
+            ?.map { it.address }
+            ?.orElse("")
+
+    private fun postnummer(postalCode: Int): String {
```

Comment:
Kan postalCode bli `null`?

Reply:
Litt usikker. Den samme koden fungerer for fastleger.